sys/usb/usbus/cdc/acm/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_USBUS_CDC_ACM
+    bool "Configure USBUS CDC ACM"
+    depends on MODULE_USBUS_CDC_ACM
+    help
+        Configure the USBUS CDC ACM module via Kconfig.
+
+if KCONFIG_MODULE_USBUS_CDC_ACM
+
+config USBUS_CDC_ACM_STDIO_BUF_SIZE
+    int "Buffer size for STDIN and STDOUT data"
+    default 128
+    depends on MODULE_STDIO_CDC_ACM
+
+config USBUS_CDC_ACM_BULK_EP_SIZE
+    int "USB CDC ACM buld endpoint size"
+    default 64
```

Comment:
Similar to the endpoint size of the previous PR, this should also be a choice of 8, 16, 32 or 64.

Reply:
Done